site/layouts/shortcodes/k8s-version.html

```diff
@@ -1,5 +1,5 @@
-{{- if eq (getenv "HUGO_ENV") "production" }}
+{{- if or (eq (getenv "HUGO_ENV") "production") (eq (getenv "HUGO_ENV") "snapshot")}}
 {{- $.Page.Site.Params.supported_k8s }}
 {{- else }}
 {{- $.Page.Site.Params.dev_supported_k8s }}
-{{- end }}
```

Comment:
Pretty sure this is caused by the inclusion of the line break to the template here. We should probably remove it.

![image](https://user-images.githubusercontent.com/298370/215219127-329a30f5-0560-440c-ba91-be02e62954c4.png)

http://localhost:1313/docs/installation/creating-cluster/minikube/